features/items.md

```diff
@@ -167,18 +184,17 @@ The output will be
     Window is 1
     Window is opened
 
-Transform file use ISO-8859-1 encoding. If you want your accent and special character to show in the right way, you need to precise this encoding in your editor. If you must use UTF-8, you will need to escape the character. See [UTF-8 encoding table](http://www.utf8-chartable.de/)
```

Comment:
Even though UTF8 characters are supported, they are maybe not by the users editor or similar. I would leave the hint to use encodings in as well?